rust/diesel/README.md

```diff
@@ -0,0 +1,21 @@
+# Build and run
+
+Build the REST API server (written using Diesel and Rocket) as follows:
+
+```
+$ cargo build --release
```

Comment:
is this also going to create the DB schema?

Reply:
Yes, the DB schema is updated/created on application startup